client/src/components/JurisdictionAdmin/RoundManagement.tsx

```diff
@@ -28,9 +28,8 @@ import { Row, Column } from '../Atoms/Layout'
 
 const Inner = styled(InnerAtom).attrs({ flexDirection: 'column' })``
 
-const SpacedDiv = styled(Card).attrs({ elevation: 1 })`
+const Panel = styled(Card).attrs({ elevation: 1 })`
   margin-bottom: 30px;
-  width: 100%;
```

Comment:
🙈

I added this for https://github.com/votingworks/arlo/pull/1661 and tested it before https://github.com/votingworks/arlo/pull/1651 was merged but not after - thank you for fixing!

Reply:
Oh interesting! I was wondering what changed. It had me quite confused

Reply:
Hmm, now I wonder why removing this line alone didn't fix it